src/mappings/TinlakeClaimRAD.ts

```diff
@@ -10,16 +10,14 @@ export function handleClaimed(claimed: Claimed): void {
 
   log.debug('handle update claim address {} to substrate address {}', [sender.toString(), centAddress.toString()])
 
-  let balance = loadOrCreateRewardBalance(sender)
+  let reward = loadOrCreateRewardBalance(sender)
   let link = loadOrCreateRewardLink(sender, centAddress)
-  balance.links = pushOrMoveLast(balance.links, link.id)
+  reward.links = pushOrMoveLast(reward.links, link.id)
 
   // add this link to their reward balance and put any
   // claimable rewards into this link, reset linkableRewards to 0
-  if (balance.claimable) {
-    link.rewardsAccumulated = link.rewardsAccumulated.plus(balance.linkableRewards)
-    balance.linkableRewards = BigDecimal.fromString('0')
-  }
-  balance.save()
+  link.rewardsAccumulated = link.rewardsAccumulated.plus(reward.linkableRewards)
+  reward.linkableRewards = BigDecimal.fromString('0')
+  reward.save()
```

Comment:
This is a breaking change, right? 

So far, all links had `0` `rewardsAccumulated` if `claimable` was `false`. By changing this, displaying data in the UI might break, and the claims admin script might allow claims even if rewards are not yet claimable.

Is this breaking change needed? Or can we stick to the old behaviour?

Reply:
ah ok - good to know about the UI
i also added this line 
`// write the linkable rewards
      lastLinked.rewardsAccumulated = lastLinked.rewardsAccumulated.plus(reward.linkableRewards)`
    which puts the linkable onto the rewards accumulated once the rewards are claimable
    which i think is sufficient
    (i added it in both places because the claim could be initiated by a user during a rewards calculation on the graph and i wasn't sure what the behavior would end up being in that case)